Failing build - cannot find Property.TSERV_WORKQ_THREADS

[My build failed](https://github.com/apache/accumulo-testing/pull/276/checks#step:5:63) with the following:
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.11.0:compile (default-compile) on project accumulo-testing: Compilation failure
[ERROR] /home/runner/work/accumulo-testing/accumulo-testing/src/main/java/org/apache/accumulo/testing/randomwalk/concurrent/Config.java:[69,59] cannot find symbol
[ERROR]   symbol:   variable TSERV_WORKQ_THREADS
[ERROR]   location: class org.apache.accumulo.core.conf.Property
```

--- a/src/main/java/org/apache/accumulo/testing/randomwalk/concurrent/Config.java
+++ b/src/main/java/org/apache/accumulo/testing/randomwalk/concurrent/Config.java
@@ -66,9 +66,6 @@ public class Config extends Test {
       Property.TSERV_COMPACTION_SERVICE_DEFAULT_EXECUTORS;
 
   @SuppressWarnings("deprecation")
-  final Property TSERV_WORKQ_THREADS_deprecated = Property.TSERV_WORKQ_THREADS;
-
-  @SuppressWarnings("deprecation")
   final Property TSERV_TABLET_SPLIT_FINDMIDPOINT_MAXOPEN_deprecated =
       Property.TSERV_TABLET_SPLIT_FINDMIDPOINT_MAXOPEN;
 
@@ -96,7 +93,6 @@ public class Config extends Test {
 			s(Property.TSERV_WAL_SORT_BUFFER_SIZE, 1024 * 1024, 1024 * 1024 * 1024L),
 			s(TSERV_TABLET_SPLIT_FINDMIDPOINT_MAXOPEN_deprecated, 5, 100),
 			s(Property.TSERV_WAL_BLOCKSIZE, 1024 * 1024,1024 * 1024 * 1024 * 10L),
-			s(TSERV_WORKQ_THREADS_deprecated, 1, 10),
 			s(Property.MANAGER_BULK_TIMEOUT, 10, 600),
 			s(Property.MANAGER_FATE_THREADPOOL_SIZE, 1, 100),
 			s(Property.MANAGER_RECOVERY_DELAY, 0, 100),

